cmd/network/verification.go

```diff
@@ -129,12 +132,33 @@ func (e *EgressVerification) Run(ctx context.Context) {
 	e.log.Info(ctx, "Preparing to check %+v subnet(s) with network verifier.", len(inputs))
 
 	for i := range inputs {
-
 		e.log.Info(ctx, "running network verifier for subnet  %+v, security group %+v", inputs[i].SubnetID, inputs[i].AWS.SecurityGroupId)
 		out := onv.ValidateEgress(c, *inputs[i])
 		out.Summary(e.Debug)
-		if out.IsSuccessful() {
```

Comment:
The call to "ValidateEgress" above will already print this out on a successful run - no need to duplicate.